lib/chef/resource/dsc_resource.rb

```diff
@@ -16,14 +16,15 @@
 # limitations under the License.
 #
 require_relative "../dsl/powershell"
+require_relative "../dist"
 
 class Chef
   class Resource
     class DscResource < Chef::Resource
       resource_name :dsc_resource
       provides :dsc_resource
 
-      description "The dsc_resource resource allows any DSC resource to be used in a Chef recipe, as well as any custom resources that have been added to your Windows PowerShell environment. Microsoft frequently adds new resources to the DSC resource collection."
+      description "The dsc_resource resource allows any DSC resource to be used in a #{Chef::Dist::PRODUCT} recipe, as well as any custom resources that have been added to your Windows PowerShell environment. Microsoft frequently adds new resources to the DSC resource collection."
```

Comment:
Could probably omit this reference.

```suggestion
      description "The dsc_resource resource allows any DSC resource to be used in a recipe, as well as any custom resources that have been added to your Windows PowerShell environment. Microsoft frequently adds new resources to the DSC resource collection."
```